applications/nrf5340_audio/src/modules/button_assignments.h

```diff
@@ -16,14 +16,21 @@
 
 #include <zephyr/drivers/gpio.h>
 
+#define BUTTON_NOT_ASSIGNED 0xFF
```

Comment:
Please remove the buttons from this PR. 
Or- create a single PR with one commit for LEDs and the other for buttons. I have not reviewed the buttons yet.

Reply:
I have to modify the buttons as the standard DK only has 4 buttons. It will fail getting the pin for button 5 (sw4).
There is a PR for the button changes, which simplifies the code, that is on top of this.